src/common/logging.cpp

```diff
@@ -49,9 +49,14 @@
 #include "common/code_utils.hpp"
 #include "common/time.hpp"
 
-static otbrLogLevel sLevel            = OTBR_LOG_INFO;
-static const char   sLevelString[][8] = {
-      "[EMERG]", "[ALERT]", "[CRIT]", "[ERR ]", "[WARN]", "[NOTE]", "[INFO]", "[DEBG]",
+static otbrLogLevel sLevel = OTBR_LOG_INFO;
+
+#if OTBR_ENABLE_DETAILED_LOGGING_FLAG
+static otbrLogLevel sDefaultLevel = OTBR_LOG_INFO;
```

Comment:
Should we set the default log level as the value passed in by command line argument `-d`? I'm kind of concerned there may be too many definitions of log level: 'detailed' log level, 'default' log level, 'start up' log level.

Reply:
It's set to the command line argument value at otbrLogInit() https://github.com/openthread/ot-br-posix/pull/1678/files#diff-ee208097f13a01b500e564c1b15f89e2184eaeb0b61f71f7bcbe684bc75709f8R94
This sDefaultLevel is used when the detailed-logging-flag changes from true to false at run time.
Another idea is to ignore the above case, and not change log level when flag changes from true to false, as this scenario should be quite rare, and a restart of otbr-agent should get things right.
Please suggest.

Reply:
Thanks. Setting default log level as the command line argument aligns with my though. I missed that line in my previous review.